srcpkgs/perl-CryptX/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'perl-CryptX'
+pkgname=perl-CryptX
+version=0.064
+revision=1
+wrksrc="${pkgname#perl-}-${version}"
```

Comment:
missing `archs=noarch`

Reply:
It does compile quite some source code and the final package contains
```/usr/lib/perl5/vendor_perl/auto/CryptX/CryptX.so```
which is arch dependent. ;)

Reply:
Maybe I've missed this when I checked the travis output.

Good then :-)

Reply:
Thanks for checking, anyway! :)